src/BylawsText.js

```diff
@@ -61,11 +82,26 @@ export default function BylawsText({ answers }) {
             <Button variant="secondary" onClick={copyBylawsToClipboard}>
               <ClipboardIcon />
               <StyledCopyText>
-                Satzungstext {isTextCopied ? 'kopiert' : 'kopieren'}
+                Satzungstext {isTextCopied ? 'wurde kopiert' : 'kopieren'}
               </StyledCopyText>
             </Button>
+            <Button
+              variant="secondary"
+              onClick={() => {
+                const text = bylawsRef.current?.innerText
```

Comment:
This looks like you get the download-text from the HTML.innertext. Is that necessary? Don't you have the ByLaws content in a state or variable somewhere?

Reply:
I am using the innertext attribute to get rid of all the html formatting within the generated Bylaws text. Also the Bylaws text consists of many variables and I don't have a single variable where everything is stored.